build.gradle.kts

```diff
@@ -24,6 +24,10 @@ dependencies {
     runtimeOnly("org.openrewrite:rewrite-java-11")
     runtimeOnly("org.openrewrite:rewrite-java-17")
 
+    implementation("org.apache.commons:commons-lang3:3.+")
+    implementation("org.codehaus.plexus:plexus-utils:3.+")
+    implementation("org.apache.maven.shared:maven-shared-utils:3.+")
```

Comment:
Looks like we could change the scope of these to `testImplementation()` or similar, right?